src/analysis/retail/warrior/arms/modules/Abilities.tsx

```diff
@@ -61,16 +61,16 @@ class Abilities extends CoreAbilities {
           suggestion: true,
           recommendedEfficiency: 0.9,
         },
-        enabled: !combatant.hasTalent(SPELLS.WARBREAKER_TALENT),
+        enabled: !combatant.hasTalent(TALENTS.WARBREAKER_TALENT),
       },
       {
-        spell: SPELLS.SKULLSPLITTER_TALENT.id,
+        spell: TALENTS.SKULLSPLITTER_TALENT.id,
         category: SPELL_CATEGORY.ROTATIONAL,
-        cooldown: (haste) => 21 / (1 + haste),
+        cooldown: (haste: number) => 22, /// (1 + haste), no longer scales with haste
```

Comment:
```suggestion
        cooldown: 22,
```
if it doesn't scale with haste, you can just set the cooldown as a number